mlshvch/hometask-5/8.sql

```diff
@@ -0,0 +1,10 @@
+DELETE FROM inventory
+WHERE inventory.film_id IN (SELECT film.film_id FROM film WHERE length > 150 );
+
+DELETE FROM film_category
+WHERE film_id IN (SELECT film.film_id FROM film WHERE length > 150 );
+
+DELETE FROM film_actor
+WHERE film_id IN (SELECT film.film_id FROM film WHERE length > 150 );
```

Comment:
I am not going to torture you with this since there isn't much time left, but the better way to write this query was to use USING option:
```suggestion
DELETE FROM film_actor USING film WHERE film_actor.film_id = film.film_id AND film.length > 120;
```

Reply:
Nothing is better than pure SQL :)